web/tests/search_suggestion.test.cjs

```diff
@@ -166,7 +166,6 @@ test("dm_suggestions", ({override, mock_template}) => {
     suggestions = get_suggestions(query);
     expected = [
         "is:dm al",
-        "is:dm is:alerted",
```

Comment:
I wonder if we want to do something for folks that have "alerted" in their muscle memory for search results. So that if they start typing 'al', there's a suggestion for 'is:watched'. We did something like this for the change from "pm-with" to "dm" operator change, I think.

Reply:
Yeah, good thought. We might want to show `is:watched` as an auto-complete result for `is:alerted` prefixes.

Reply:
Probably not a lot of downside, as there's unlikely to be a different reason to type `is:al`.